### PR TITLE
fix: displays price when not updated on TDP

### PR DESCRIPTION
**Pull request by LunrEclipse**
## Description

Displays that price as opposed to `Price unavailable` with a tooltip informing users that the price is not updated in the event where there is some price data, but it may not be up to date. 

_Linear ticket:_ WEB-2558

### Before

Screenshot 2023-08-02 at 3 26 01 PM

Screenshot 2023-08-02 at 3 25 04 PM

### After

Screenshot 2023-08-02 at 3 26 26 PM

Screenshot 2023-08-02 at 3 24 19 PM
Screenshot 2023-08-02 at 3 24 40 PM

### Automated testing

- [x] Unit test N/A

### CODE REVIEW

**Review by tinaszheng — APPROVED** [review]
Nice!

File: src/components/Tokens/TokenDetails/PriceChart.tsx [hunk 1]
```diff
@@ -148,6 +175,34 @@ export function PriceChart({ width, height, prices: originalPrices, timePeriod }
     )
   ) : null
 
+  const tooltipMessage = (
+    <>
+      <Trans>Price may not be up-to-date due to recently low trading volume</Trans>
```
Comment (JFrankfurt) [inline src/components/Tokens/TokenDetails/PriceChart.tsx:180]: This reads quite awkwardly imo.

```suggestion
      <Trans>This price may not be up-to-date due to low trading volume.</Trans>
```

**Review by JFrankfurt — CHANGES_REQUESTED** [review]
Add a unit test and this'll be good to go.

**Review by JFrankfurt — APPROVED** [review]
noice

**Pull request merged**